docs/release-notes/cloud/cloud-2023-06.md

```diff
@@ -0,0 +1,74 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud June 2023.
+included_jira_versions: ['2023.06']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.5.4
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.5.46
+---
+
+# Cloud June 2023
+
+These release notes are for the Codacy Cloud updates during June 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   There's now an onboarding tutorial to guide you through the steps of [adding and setting up your first repository](../../getting-started/codacy-quickstart.md#adding-your-first-repository). (DOCS-468)
+
+## Bug fixes
+
+-   While [configuring code patterns](../../repositories-configure/configuring-code-patterns.md), it's now possible to correctly configure rules using strings that include the character sequence `\\`, such as regular expressions. Before, Codacy stripped out one of the `\` characters. (IO-484)
```

Comment:
Hi @betullkoyuncu! Could you please check if this release note for the issue [IO-484](https://codacy.atlassian.net/browse/IO-484) is correct? Thanks!